### DISCUSSION TITLE
Aporte extensiones vscode

### DISCUSSION THREAD

**Discussion by Syara224** [Launch-X-Latam/MisionFrontEnd#183] [Ideas]
Hola a todos, les recomiendo tener en su vs code las siguientes extensiones que les pueden facilitar la vida y el desarrollo de las  actividades.

prettier
live server
Auto Rename Tag
Color Highlight

**Comment by doguedogue** [comment]
genial gracias, echale un ojo a VS Code PETS 🐶

**Comment by AndrjsM** [comment]
Gracias buena persona
